sources/packages/api/src/auth/guards/groups.guard.ts

```diff
@@ -33,8 +33,9 @@ export class GroupsGuard implements CanActivate {
     }
     const userToken = user as IUserToken;
     // Check if the user has any of the groups required to have access to the resource.
+    // UserGroups 'aest/user/x' and 'aest/user' are valid, we are here looking for matches, not the exact string.
```

Comment:
Comment can also mention that both the groups are valid as they start with 'aest'.